Content.Shared/Weapons/Melee/Components/EnergyShieldComponent.cs

```diff
@@ -0,0 +1,43 @@
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Weapons.Melee.EnergyShield;
+
+[RegisterComponent, NetworkedComponent]
+public sealed class ItemToggleComponent : Component
+{
+    public bool Activated = false;
+
+    [DataField("isSharp")]
+    public bool IsSharp = true;
```

Comment:
```suggestion
```

Shields aren't sharp. Also if they were, they shouldn't default to being sharp in the off state.

Reply:
removed